public/jsx/components/card-form.jsx

```diff
@@ -98,6 +110,39 @@ module.exports = React.createClass({
     this.setState(stateChange);
   },
 
+  handleSubmit: function(e) {
+    var { router } = this.context;
+    e.preventDefault();
+    this.setState({isSubmitting: true});
+    var client = new braintree.api.Client({
+      clientToken: this.props['data-token'],
+    });
+    client.tokenizeCard({
+      number: this.state.card,
+      expirationDate: this.state.expiration,
+      cvv: this.state.cvv,
+      cardholderName: this.state.cardholder,
+    }, function(err, nonce) {
+      if (err) {
+        // TODO: error handling
+        console.log(err);
+      } else {
+        $.ajax({
+          data: {
+            pay_method_nonce: nonce,
+            plan_id: 'mozilla-concrete-brick',
+          },
+          url: '/api/braintree/subscriptions/',
+          method: 'post',
+          dataType: 'json',
+        }).then(function(subscribeData) {
+          console.log('successfully subscribed?', subscribeData);
```

Comment:
As far as I know this response won't have any content (unless we think of something) so the log message could just be generic.